### DISCUSSION TITLE
Model size of pre-trained models

### DISCUSSION THREAD

**Discussion by Tirthankar-iiitb** [espnet/espnet#4272] [Q&A]
How to find model parameters of a pre-trained ESPnet ASR model from model zoo? For example, the SPGIspeech trained model - Shinji_Watanabe/spgispeech_asr_train_asr_conformer6_n_fft512_hop_length256_raw_en_unnorm_bpe5000_valid.acc.ave using ModelDownloader.download_and_unpack(tag)?

**Comment by emmastoneviral23** [comment]
You can find the parameters used to train an ASR model in the corresponding README file in the model zoo. The most important parameter is the embedding dimension (D[)](https://heatcaster.com/). The embedding dimension is the number of unique frames that the model is trained on[.](https://fxpfitness.com/) A larger embedding dimension will generally improve the model's performance, but will require more time to train. You can check the performance of many models in the model zoo on the model comparison page[.](https://lettherebemovies.com/multi-channel-home-theatre/)

**Reply by emmastoneviral23** [reply]
The model might be in C, C[+](https://healthmantion.com/)[+](https://techmantion.com/), or Python. There are also open-source implementations of models in Torch (don't know if they come with pretrained parameters).

**Reply by pakkaujala2** [reply]
How to implement the code in C and C[+](https://techmantion.com/best-embroidery-machine-for-custom-designs/)[+](https://heatcaster.com/attitude-status/)[?](https://healthmantion.com/can-you-get-in-shape-with-a-power-tower/) can you please guide me?
